style/Typography.css

```diff
@@ -0,0 +1,42 @@
+.Typo-items{
+    padding:20px;
+    border: 0.5px solid var(--border-color);
+    margin: 20px;
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 350px;
+    font-weight: 600px;
```

Comment:
this won't work, remove px from font weight

Reply:
oh thanks that was by mistake